fix(issue#25182): eventhub and svervicebus processor client start span independently (#26180)
Azure/azure-sdk-for-java: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

# Release History

## 5.10.5-beta.1 (Unreleased)

### Bugs Fixed

> - Issue if received message does not have trace context, spans are not created ([#25182](https://github.com/Azure/azure-sdk-for-java/issues/25182))

Reviewer: ```suggestion
- Fixed a bug that when received message does not have trace context, span is not created. ([#25182](https://github.com/Azure/azure-sdk-for-java/issues/25182))
```

Reviewer: usually you can take core as example
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/CHANGELOG.md

Author: > usually you can take core as example https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/CHANGELOG.md

Thanks. I refer to the example on this page https://azure.github.io/azure-sdk/policies_releases.html#change-logs
